e2e/common/common.steps.ts

```diff
@@ -86,6 +102,27 @@ class CommonSteps {
       .to.eventually.be.true.notify(callback);
   }
 
+  @then(/^she is presented with the "([^"]*)" elements*$/)
+  public expectElementsPresent(elementClassNames: string, callback: CallbackStepDefinition): void {
+
+    const elementClassNamesArray = elementClassNames.split(',');
```

Comment:
Can we find another way of implementing this step?

```gherkin
# I really like this lavel ob abstraction - it seems pretty readable
When Camilla clicks on the "View All Integrations" link

# defining dom elements or css classes should remain in page objects imho
Then she is presented with the "top-integrations,integration-board,recent-updates,ipaas-dashboard-connections" elements
```

What do you think @lvydra @dsimansk  ?